plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -207,6 +214,17 @@ private function get_feature_definitions() {
 					'disable_ui'         => false,
 					'is_legacy'          => true,
 				),
+				'order_source_attribution' => array(
+					'name'               => __( 'Order Source Attribution', 'woocommerce' ),
+					'description'        => __(
+						'Enable this feature to track and credit channels and campaigns that contribute to orders on your site. (Beta)',
```

Comment:
💅🏻 Most existing settings labels seem to omit the `.`:
```suggestion
						'Enable this feature to track and credit channels and campaigns that contribute to orders on your site (Beta)',
```
![image](https://github.com/woocommerce/woocommerce/assets/228780/d3854275-402e-4fb4-a7d3-59b26133a748)

Reply:
❔ Also, by definition of the feature being beta, should it be in the **Experimental features** section? Although I understand that it may be alarming to find an Experimental feature  auto-enabled ¯\_(ツ)_/¯

Reply:
> Also, by definition of the feature being beta, should it be in the Experimental features section? Although I understand that it may be alarming to find an Experimental feature auto-enabled ¯_(ツ)_/¯

We talked about it, and decided that it even though it's beta, it doesn't fit with the warning given in Experimental features being something to "enable [...] at your own risk!"

pe2C5g-JD-p2#comment-775